targets/nano-33-ble-sense-s140v7.json

```diff
@@ -0,0 +1,3 @@
+{
+	"inherits": ["nano-33-ble-s140v7"]
```

Comment:
Removed

Reply:
Oh - now I see. Because there was also a `nano-33-ble-sense`.

Reply:
Once we have comments in target files, we could add info in “nano-33-ble” that it shall be used for “sense” variant too… so we skip an extra almost empty file. Or you prefer I add them back, since I’ve removed them now.

Reply:
I'm fine either way. But if you add the `nano-33-ble-sense.json` file back, then there should also be such an alias for the -s140v7 variant.